runelite-client/src/main/java/net/runelite/client/util/LinkBrowser.java

```diff
@@ -48,37 +49,68 @@
 	/**
```

Comment:
Change class javadoc to read `Utility class used for web and file browser navigation`